### PR TITLE
Added integration testing script

### PR DESCRIPTION
**Pull request by faraonc**
Integration testing script added for hwsc-file-metadata-svc. Added callback checking.

### CODE REVIEW

File: int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js [hunk 1]
```diff
@@ -12,6 +12,11 @@ const hwscFileMetadataSvcProtoPkgDef = protoLoader.loadSync("", options);
 const hwscFileMetadataSvcPbJs = grpc.loadPackageDefinition(hwscFileMetadataSvcProtoPkgDef).hwscFileMetadataSvc;
 
 function getStatus(callback) {
+    if (typeof callback !== "function") {
```
Comment (kimlisa) [inline int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: Should we make a function that does callback checking to remove duplicated console.error messages?

Comment (kimlisa) [reply int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: but then again its a testing script, so maybe it doesn't matter :3

File: int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js [hunk 2]
```diff
@@ -25,6 +30,11 @@ function getStatus(callback) {
 }
 
 function createFileMetadata(fileMetadata, callback) {
+    if (typeof callback !== "function") {
+        console.error("callback not a function");
+        return;
+    }
+
     const client = new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
```
Comment (kimlisa) [inline int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: const for localhost:50051?

Comment (kimlisa) [reply int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: or function that returns new client connection?

```
function createClient() {
   return new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
        grpc.credentials.createInsecure());
}
```

Comment (kimlisa) [reply int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: I also see this a lot too:
```javascript
function (err, response) {
        if (!err) {
            console.log(request);
            grpc.closeClient(client);
        }

        callback(err, response);
    });
```

Will this work?

```javascript
client.createFileMetadata(request, (err, response) => processResponse(err, response, callback);

function processResponse(err, response, callback) {
    if (!err) {
      console.log(request);
      grpc.closeClient(client);
    }

    callback(err, response);
}
```

File: test_file_metadata_svc_client.js [hunk 3]
```diff
@@ -90,23 +93,33 @@ const dataSet = [
...
-    index.hwscFileMetadataSvc.listFileMetadataCollection(dataSet[parseInt(process.argv[2])], callback)
+    switch(process.argv[2]){
+        case "0":
+            index.hwscFileMetadataSvc.getStatus(callback);
+            break;
+        case "1":
+            index.hwscFileMetadataSvc.createFileMetadata(dataSet[parseInt(process.argv[3])], callback);
+            break;
+        case "2":
+            index.hwscFileMetadataSvc.listFileMetadataCollection(dataSet[parseInt(process.argv[3])], callback)
```
Comment (kimlisa) [inline test_file_metadata_svc_client.js]: missing semicolon

**Comment by kimlisa** [comment]
TU

File: int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js (hunk 2 above)
Comment (faraonc) [reply int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: > const for localhost:50051?

we will start a ticket for config file

Comment (faraonc) [reply int/hwsc-file-metadata-svc/hwsc-file-metadata-svc.js]: > and function that returns new client connection? Will that even work?
> 
> ```js
> function createClient() {
>    return new hwscFileMetadataSvcPbJs.FileMetadataService("localhost:50051",
>         grpc.credentials.createInsecure());
> }
> ```

we will start a ticket

**Pull request merged**